src/gwt/src/org/rstudio/studio/client/workbench/views/connections/ui/ObjectBrowser.java

```diff
@@ -69,80 +69,21 @@ public void clear()
    
    public void update(Connection connection, String hint)
    { 
-      final Set<DatabaseObject> expandedNodes = new HashSet<DatabaseObject>();
-      
-      // if this update is for the currently visible connection in the model,
-      // cache the set of expanded nodes for replay
-      if (objects_ != null && connection == connection_)
-      {
-         TreeNode rootNode = objects_.getRootTreeNode();
-         if (!objects_.getRootTreeNode().isDestroyed())
-         {
-            for (int i = 0; i < rootNode.getChildCount(); i++)
-            {
-               if (rootNode.isChildOpen(i))
-               {
-                  DatabaseObject node = (DatabaseObject)rootNode.getChildValue(i);
-                  expandedNodes.add(node);
-               }
-            }
-         }
-      }
-      
       // create tables model and widget
       objectsModel_ = new ObjectBrowserModel();
       
-      // capture scroll position
-      final int scrollPosition = scrollPanel_.getVerticalScrollPosition();
-
       // show progress while updating the connection
       hostPanel_.showProgress(50, "Loading objects");
             
       // update the table then restore expanded nodes
       objectsModel_.update(
-         connection,      // connection 
-         expandedNodes,
-         new Command() {   // table update completed, expand nodes
-            @Override
-            public void execute()
-            {
-               // clear progress and show the object tree again
-               hostPanel_.setWidget(scrollPanel_);
-               
-               // restore expanded nodes
-               TreeNode rootNode = objects_.getRootTreeNode();
-               if (!rootNode.isDestroyed())
-               {
-                  for (int i = 0; i < rootNode.getChildCount(); i++)
-                  {
-                     final DatabaseObject nodeVal = 
-                           (DatabaseObject)(rootNode.getChildValue(i));
-                     for (DatabaseObject expanded: expandedNodes)
-                     {
-                        if (expanded.isEqualTo(nodeVal))
-                           rootNode.setChildOpen(i, true, false);
-                     }
-                  }
-               }
-            }
-         },
-         new Command() {   // node expansion completed, restore scroll position
-            @Override
-            public void execute()
-            {
-               // delay 100ms to allow expand animation to complete
-               new Timer() {
-
-                  @Override
-                  public void run()
-                  {
-                     scrollPanel_.setVerticalScrollPosition(scrollPosition); 
-                  }
-                  
-               }.schedule(100);
-              
-            }
-         });
+         connection ,      // connection 
+         null,             // expanded nodes (none for refresh)
+         () -> 
```

Comment:
Noticed you put opening brace on a new line for the lambda.

I think this does match the style guide (which doesn't specifically mention lambdas, so the function braces-on-newline would apply), but our Java and C++ code has a mix of both styles.

I've been using opening brace on same line for these, either because the first example I found was doing that or because my editor's default config does that... We should pick one and add it to the style guide and gradually switch over offenders.

I don't personally have a preference either way, but like not having to think about it.

Reply:
You're right, we should pick one and be consistent! I'll make a note.